covasim/webapp/index.html

```diff
@@ -36,199 +36,222 @@
   <header class="row no-gutter">
     <b-navbar type="dark" variant="dark">
       <b-navbar-brand href="#">{{ app.title }}</b-navbar-brand>
-      <b-navbar-nav>
-        <b-nav-item :href="app.docs_url" target="_blank">Documentation</b-nav-item>
-        <b-nav-item v-b-modal.about>About</b-nav-item>
+      <b-navbar-nav class="nav-head">
+        <b-nav-item :href="app.docs_url" target="_blank"><span class="nav-text">Documentation</span></b-nav-item>
+        <b-nav-item v-b-modal.about><span class="nav-text">About</span></b-nav-item>
       </b-navbar-nav>
       <b-navbar-nav class="ml-auto">
-        <b-nav-item :href="app.github_url" target="_blank">{{app.version}}</b-nav-item>
+        <b-nav-item :href="app.github_url" target="_blank"><span class="nav-text">{{app.version}}</span></b-nav-item>
       </b-navbar-nav>
     </b-navbar>
```

Comment:
Overriding the nav-text class and adding the spans is unnecessary. Instead, just override the nav-link class.

Reply:
Added to issue